src/main/java/seedu/address/model/person/Doctor.java

```diff
@@ -0,0 +1,90 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents a Doctor class in the Doctors Records
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Doctor extends Person {
+    // Doctor contact fields
+    private final Phone phone;
+    private final Email email;
+    private final Address address;
+
```

Comment:
We discussed that we will not need to maintain the email or address of the doctor.